frontend/src/metabase-types/api/dataset.ts

```diff
@@ -72,6 +72,22 @@ export interface Dataset {
   status?: string;
 }
 
+export interface PublicDatasetData {
+  rows: RowValues[];
+  cols: DatasetColumn[];
+  rows_truncated: number;
+  // TODO: Correct this type
```

Comment:
Discussed with @NevRA and it's fine to leave this as is. The problem is that it's really hard to add a new type in FE at the moment because BE types are hard to find. There isn't an easy-to-read interface on the BE that says this API will return this shape or something like that, so I need to resort to reading the BE implementation to figure out what fields each endpoint return. And since we don't care about `insights` here, it should be fine. I've included it for the sake of completeness.